liquidity_pool/src/liquidity.rs

```diff
@@ -182,11 +180,11 @@ pub trait LiquidityModule:
         Ok(())
     }
 
-    // Returns the asset token ID, the amount of debt paid
     #[only_owner]
     #[payable("*")]
     #[endpoint]
     fn repay(&self, initial_caller: Address) -> SCResult<()> {
+        let lend_token_amount_to_send_back: Self::BigUint;
```

Comment:
There's little reason to declare this here. I think it's best declared before it's assigned to, i.e. line ~232

Reply:
Done